build.sc

```diff
@@ -189,18 +191,18 @@ trait MillMimaConfig extends mima.Mima {
     main.core -> Seq(
       ProblemFilter.exclude[IncompatibleSignatureProblem]("mill.define.Target.makeT"),
       ProblemFilter.exclude[IncompatibleSignatureProblem]("mill.define.Target.args"),
-      ProblemFilter.exclude[IncompatibleSignatureProblem]("mill.util.ParseArgs.standaloneIdent"),
+      ProblemFilter.exclude[IncompatibleSignatureProblem]("mill.define.ParseArgs.standaloneIdent"),
       ProblemFilter.exclude[IncompatibleSignatureProblem](
-        "mill.util.ParseArgs#BraceExpansionParser.plainChars"
+        "mill.define.ParseArgs#BraceExpansionParser.plainChars"
       ),
       ProblemFilter.exclude[IncompatibleSignatureProblem](
-        "mill.util.ParseArgs#BraceExpansionParser.braceParser"
+        "mill.define.ParseArgs#BraceExpansionParser.braceParser"
       ),
       ProblemFilter.exclude[IncompatibleSignatureProblem](
-        "mill.util.ParseArgs#BraceExpansionParser.parser"
+        "mill.define.ParseArgs#BraceExpansionParser.parser"
       ),
       ProblemFilter.exclude[IncompatibleSignatureProblem](
-        "mill.util.ParseArgs#BraceExpansionParser.toExpand"
+        "mill.define.ParseArgs#BraceExpansionParser.toExpand"
```

Comment:
Hm, these don't make any sense as these are completely new (under this fqn).

Reply:
Yeah but if we were excluding it under the old FQN name in the past, we presumably want to exclude it under the new FQN in the future don't we?

Reply:
Right. What I really meant, MiMa will probably not report these issue at all, as the classes/traits have a new name. So the filters are rather meaningless.